lib/tasks/scheduler.rake

```diff
@@ -12,7 +14,7 @@ task house_valuation_collector: :environment do
     end
   end
 
-  HappyHood::Slack::Client.send_daily_price_summary
+  Rails.logger.info { "Completed getting valuation in #{Time.now - start_time } seconds" }
```

Comment:
Added timing information to try to start getting an idea of how long this stuff is taking us.